[14.0][FIX] incompatible modules

nilshamerlinck opened the pull request:
`sale_isolated_quotation` and `sale_quotation_number` are incompatible (both define a sequence with same code `sale.quotation`): they should not be installed nor tested together

nilshamerlinck committed:
[FIX] sale_isolated_quotation and sale_quotation_number are incompatible: they should not be installed nor tested together

simahawk commented on .travis.yml:
```diff
@@ -20,10 +20,10 @@ jobs:
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE="sale_quotation_number"
```
is this relevant? It's only for generating POT no?

pedrobaeza replied:
It makes both things: test with `odoo/odoo` and generate the POT, but anyways, you need to generate POT for all modules, so each EXCLUDE/INCLUDE should have both lines.

nilshamerlinck replied:
right, fixed

simahawk commented on .travis.yml:
```diff
@@ -20,10 +20,10 @@ jobs:
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="OCA/OCB"
+        - TESTS=1 ODOO_REPO="OCA/OCB" EXCLUDE="sale_isolated_quotation"
```
don't we need one that excludes `sale_quotation_number`?

simahawk replied:
I would still change the name of the sequence and avoid running tests separately

nilshamerlinck replied:
but beyond the sequence, they are overlapping functionally so testing them together could lead to other issues in the future

nilshamerlinck committed:
[IMP] Manage incompatible modules through copier update

- sale_partner_version was already tested separately since 9b743551 because of its incompatibility with sale_tier_validation; just explicitely added to rebel_module_groups
- sale_quotation_number must also be tested separately because of its incompatibility with sale_isolated_quotation

nilshamerlinck commented:
Hi @simahawk @pedrobaeza
- I updated the PR to properly use the _rebel_module_groups_ setting of the repo template
- we are being bitten by this on some PRs again, would you mind merging it?

simahawk commented:
/ocabot merge patch
